### PR TITLE
Add precondition of importing Harvester to Rancher

### PR DESCRIPTION
**Pull request by w13915984028**
Issue: https://github.com/harvester/harvester/issues/5658

### CODE REVIEW

File: docs/rancher/virtualization-management.md [hunk 1]
```diff
@@ -32,12 +32,36 @@ Please refer to the [Harvester & Rancher Support Matrix](https://www.suse.com/su
...
+
+    - Set a private [containerd-registry](../advanced/settings.md#containerd-registry) and add the additional image. Harvester will pull the image from this registry automatically.
+
+    - If [http-proxy](../airgap.md/#configure-an-http-proxy-in-harvester-settings) is used, configure and confirm it is working as expected. Your configured [DNS servers](../install/update-harvester-configuration.md#dns-servers) should be able to resolve the domain name `docker.io`.
+
+    - Manually download the image (`docker pull rancher/rancher-agent:v2.7.9 && docker save -o rancher-agent.img rancher/rancher-agent:v2.7.9`), copy it to each node of your Harvester cluster and load it (`sudo -i docker image load -i rancher-agent.img`). Run `crictl image ls | grep "rancher-agent"` on each node to ensure the image is ready.
```
Comment (bk201) [inline docs/rancher/virtualization-management.md:43]: I'd suggest using `ctr` command for loading images in containerd. `docker` is an alias to `nertdctl` and `nerdctl` is included just for convenience, it's not officially supported by SUSE or RKE2 team.

Comment (bk201) [reply docs/rancher/virtualization-management.md]: Thanks @w13915984028. one tiny nit. Can we name `rancher-agent.img` to `rancher-agent.tar`? it's a tarball.

**Review by starbops — APPROVED** [review]
LGTM, thank you!

**Review by jillian-maroket — COMMENTED** [review]
Review done

File: docs/rancher/virtualization-management.md [hunk 2]
```diff
@@ -32,12 +32,36 @@ Please refer to the [Harvester & Rancher Support Matrix](https://www.suse.com/su
...
 1. Once the Rancher server is up and running, log in and click the hamburger menu and choose the **Virtualization Management** tab. Select **Import Existing** to import the downstream Harvester cluster into the Rancher server.
 ![](/img/v1.2/rancher/vm-menu.png)
+
 1. Specify the `Cluster Name` and click **Create**. You will then see the registration guide; please open the dashboard of the target Harvester cluster and follow the guide accordingly.
 ![](/img/v1.2/rancher/harv-importing.png)
+
 1. Once the agent node is ready, you should be able to view and access the imported Harvester cluster from the Rancher server and manage your VMs accordingly.
 ![](/img/v1.2/rancher/harv-cluster-view.png)
+
+  When the agent node is stucking, run `kubectl get pod cattle-cluster-agent-*** -n cattle-system -oyaml` on Harvester cluster. If the below message is observed, re-check the above step 1 then kill this pod to retry.
```
[the remark this answers is not in the document]
Comment (w13915984028) [reply docs/rancher/virtualization-management.md]: Changed a bit for sentence 
`kill this pod, and restart the import process` 
to 
`kill this pod and then a new pod will be created automatically to restart the importing process`. 
The restart is done by k8s automatically, not the user.

**Review by jillian-maroket — APPROVED** [review]

**Pull request merged**